src/components/Article.jsx

```diff
@@ -15,6 +17,30 @@ function Article() {
     });
   }, []);
 
+  const handleClickVoteUp = () => {
+    patchArticle(article_id, 1)
+      .then((updatedArticle) => {
+        setArticleVotes((currentArticleVotes) => {
+          return currentArticleVotes + 1;
+        });
+      })
+      .catch((err) => {
+        setApiError(err.response);
+      });
+  };
+
+  const handleClickVoteDown = () => {
+    patchArticle(article_id, -1)
+      .then((updatedArticle) => {
+        setArticleVotes((currentArticleVotes) => {
+          return currentArticleVotes - 1;
+        });
+      })
+      .catch((err) => {
+        setApiError(err.response);
+      });
```

Comment:
we could make this a little more DRY - as these two functions are doing the same thing it's just whether it's plus or minus 1 - think how you could just have one function!